src/Xamarin.Android.Build.Tasks/Tasks/NdkUtils.cs

```diff
@@ -25,18 +25,21 @@ public static class NdkUtil
 
 		public static bool Init (string ndkPath)
 		{
-			return Init (null, ndkPath); // For tests which don't have access to a TaskLoggingHelper
+			return Init (delegate { }, ndkPath); // For tests which don't have access to a TaskLoggingHelper
 		}
 
-		public static bool Init (TaskLoggingHelper log, string ndkPath)
+		public static bool Init (TaskLoggingHelper log, string ndkPath) =>
```

Comment:
I assume this the the old method that should not really be used? Should we add an Obsolete attribute?

Reply:
This method was still used by the task that runs `mkbundle`. It is not `AsyncTask`, so it is OK to use this method.

With the change to add `[Obsolete(error:true)]` we would still get a compiler error if we tried to pass `AndroidAsyncTask.Log`.